postman/Get Clicks.postman_collection.json

```diff
@@ -0,0 +1,577 @@
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"method": "GET",
+						"header": [],
+						"body": {
+							"mode": "raw",
+							"raw": ""
+						},
+						"url": {
+							"raw": "{{base_URL}}/accounts/{{accountId}}/lists",
+							"host": [
+								"{{base_URL}}"
+							],
+							"path": [
+								"accounts",
+								"{{accountId}}",
+								"lists"
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "Get Random List",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"id": "b41770d3-1044-48f5-9536-6cf9ba89088a",
+								"exec": [
+									"pm.test(\"Sucessfully got a random list\", function() {",
+									"    var jsonData = pm.response.json();",
+									"    pm.expect(jsonData.total_size).to.be.a('number')",
+									"    var listId = Math.floor((Math.random() * jsonData.total_size ))",
+									"    pm.globals.set(\"listId\",jsonData.entries[listId].id )",
+									"});",
+									""
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"method": "GET",
+						"header": [],
+						"body": {
+							"mode": "raw",
+							"raw": ""
+						},
+						"url": {
+							"raw": "https://api.aweber.com/1.0/accounts/{{accountId}}/lists",
+							"protocol": "https",
+							"host": [
+								"api",
+								"aweber",
+								"com"
+							],
+							"path": [
+								"1.0",
+								"accounts",
+								"{{accountId}}",
+								"lists"
+							]
+						}
+					},
+					"response": []
+				}
+			]
+		},
+		{
+			"name": "Get Broadcast Open Clicks",
+			"item": [
+				{
+					"name": "Get Total Sent Broadcasts",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"id": "e238f70c-3010-41a2-ba60-6fd4e312850d",
+								"exec": [
+									"pm.test(\"Sucessfully got total number of broadcasts\", function(){",
+									"    var jsonData = pm.response.json();",
+									"    var totalBroadcasts = Math.floor((Math.random() * jsonData.total_size ))",
+									"    pm.globals.set(\"totalBroadcasts\",totalBroadcasts )",
+									"})",
+									""
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"method": "GET",
+						"header": [],
+						"body": {
+							"mode": "raw",
+							"raw": ""
+						},
+						"url": {
+							"raw": "https://api.aweber.com/1.0/accounts/{{accountId}}/lists/{{listId}}/broadcasts/total?status=sent",
+							"protocol": "https",
+							"host": [
+								"api",
+								"aweber",
+								"com"
+							],
+							"path": [
+								"1.0",
+								"accounts",
+								"{{accountId}}",
+								"lists",
+								"{{listId}}",
+								"broadcasts",
+								"total"
+							],
+							"query": [
+								{
+									"key": "status",
+									"value": "sent"
+								}
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "Get Sent Broadcasts",
```

Comment:
Postman has a real problem generating the OAuth signature for this request and I'm not sure why. I can only infrequently get it to work.

Reply:
<img width="984" alt="screen shot 2018-11-29 at 9 37 41 am" src="https://user-images.githubusercontent.com/3092064/49228973-8a9fdf80-f3ba-11e8-8396-555714e50402.png">